Wendy.jiang/update event management doc (#22900)
DataDog/documentation: content/en/service_management/events/correlation/configuration.md

There are two types of correlations: 

> - **Pattern-based**: you control how the events/alerts are correlated, though the information available within them. Datadog also automatically enriches your pattern based correlation with intelligent alerts powered by ML model.

Reviewer: ```suggestion
- **Pattern-based**: you control how the events or alerts are correlated. Datadog also automatically enriches your pattern based correlation with intelligent alerts powered by ML model.
```
